bin/src/command/orders.rs

```diff
@@ -599,6 +599,25 @@ impl CommandServer {
       ::std::env::set_var("RUST_LOG", logging_filter);
     }
 
+    // Check if the backend or frontend exist before deleting it
+    match order {
```

Comment:
if `worker_id` is not `None`, we should not return from here: the message is specifically addressed to one worker. It is useful if an external tool detects that the master process and the worker do not have the same configuration.